tools/sdk-generation-pipeline/packages/sdk-generation-cli/src/publishResultCli.ts

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env node
+import * as fs from 'fs';
+import { Connection, createConnection } from 'typeorm';
+
+import {
+    AzureBlobClient,
+    CodeGeneration,
+    logger,
+    publishResultConfig,
+    PublishResultConfig,
+    uploadArtifacts,
+    uploadSourceCode,
+    uploadLogsAndResult,
+    requireJsonc,
+    sendSdkGenerationToDB,
+    sendSdkTaskResultToDB,
+    TaskResult,
+    TaskResultEntity,
+    GenerateAndBuildOutput,
+    getGenerateAndBuildOutput,
+} from '@azure-tools/sdk-generation-lib';
+
+async function SendDB(
+    taskResultsPath: string,
+    pipelineBuildId: string,
+    sdkGenerationName: string,
+    service: string,
+    serviceType: string,
+    language: string,
+    swaggerRepo: string,
+    sdkRepo: string,
+    codegenRepo: string,
+    triggerType: string,
+    status: string,
+) {
```

Comment:
I heard today that other team are also using Kusto for data storage, so why do we still use mongo db?

Reply:
I would like to ask an even more generic question which is why are writing anything to a database here? Where do we expect that information to be consumed or used? If our goal is ultimately to use an Azure DevOps pipeline then perhaps we just upload logs as artifacts to that pipeline instead of trying to maintain state somewhere else.

Reply:
> I would like to ask an even more generic question which is why are writing anything to a database here? Where do we expect that information to be consumed or used? If our goal is ultimately to use an Azure DevOps pipeline then perhaps we just upload logs as artifacts to that pipeline instead of trying to maintain state somewhere else.

Hi Wes, these variable are all the info for one sdk generation build pipeline. While PowerApps's flow  want to inquire a pipeline's state or result. Then it can send a request to the DB.

Reply:
Why would PowersApps directly query a DB as opposed to just querying DevOps? 

I've also asked @chunyu3 to schedule a meeting about some of the design decisions around this flow, I think you are on the email and you can read some of my design feedback in the document.

Reply:
thx Wes, I'm looking your feedback. We will discuss it in the meeting.